tests/test_dvc.py

```diff
@@ -185,3 +188,106 @@ def test_exp_save_dvcexception_is_ignored(tmp_dir, mocker):
 
     with Live(save_dvc_exp=True):
         pass
+
+
+def test_get_dvc_stage_template_empty(tmp_dir, mocked_dvc_repo):
+    live = Live()
+    template = get_dvc_stage_template(live)
+
+    assert YAML_LOADER.load(template) == {
+        "stages": {
+            "dvclive": {
+                "cmd": "<python my_code_file.py my_args>",
+                "deps": ["<my_code_file.py>"],
+                "outs": [],
+            }
+        }
+    }
+
+
+def test_get_dvc_stage_template_params(tmp_dir, mocked_dvc_repo):
+    mocked_dvc_repo.root_dir = tmp_dir
```

Comment:
Is this needed?

Reply:
Without I get an assertion error like:

```
E         Full diff:
E           {
E            'stages': {'dvclive': {'cmd': '<python my_code_file.py my_args>',
E                                   'deps': ['<my_code_file.py>'],
E         -                         'outs': [{'dvclive/params.yaml': {'cache': False}}]}},
E         +                         'outs': [{'../../../dvclive/params.yaml': {'cache': False}}]}},
E         ?                                    +++++++++
E           }
```

Reply:
Ok, that looks bad 😅 I need to fix the fixtures

Reply:
I can put something like `_dvc_repo.root_dir = "."` into https://github.com/iterative/dvclive/blob/main/tests/conftest.py#L20 if you prefer.

Reply:
I think that maybe `mocked_dvc_repo` should not be a fixture but a function you call passing the `tmp_dir` fixture you have on each test

Reply:
Adding to the fixture like I mention above does work. Was your concern about that or more about something else like style or correctness?